tests/chainer_tests/utils_tests/test_sparse.py

```diff
@@ -54,4 +54,58 @@ def test_to_dense(self):
         numpy.testing.assert_array_equal(x0, x1)
 
 
+@testing.parameterize(*testing.product({
+    'shape': [(2, 3), (3, 4)],
+    'nbatch': [0, 1, 4],
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestGetOrder(unittest.TestCase):
+
+    def test_get_order(self):
+        if self.nbatch > 0:
+            x_shape = (self.nbatch, self.shape[0], self.shape[1])
+        else:
+            x_shape = self.shape
+        x0 = _setup_tensor(.5, 1, x_shape, self.dtype, .75)
+        x0 = numpy.ascontiguousarray(x0)
+        if self.nbatch > 0:
+            x0[0, 0, self.shape[1]-1] = 1.
+            x0[0, self.shape[0]-1, 0] = 1.
+        else:
+            x0[0, self.shape[1]-1] = 1.
+            x0[self.shape[0]-1, 0] = 1.
+        sp_x = utils.to_coo(x0)
+        row = sp_x.row
+        col = sp_x.col
+        assert utils.get_order(row, col) == 'C'
+        assert utils.get_order(col, row) == 'F'
+
+
+class TestGetOrder2(unittest.TestCase):
```

Comment:
Should we add tests with `F` order matrices?

Reply:
I think that is already tested in `TestGetOrder`.